scripts/upgrade/authelia.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+if [[ ! -f /install/.authelia.lock ]]; then
+    echo_warn "Authelia is not installed"
+    exit 1
+else
+    echo_progress_start "Updating Authelia"
+    systemctl stop -q authelia |& tee -a $log
+    # Set the current version using a git ls-remote tag check
+    authelia_latestv="$(git ls-remote -t --sort=-v:refname --refs https://github.com/authelia/authelia.git | awk '{sub("refs/tags/", "");sub("(.*)-alpha(.*)", ""); print $2 }' | head -n1)"
```

Comment:
Any chance we can check if the upgrade is necessary or not?

Reply:
how else will it get updated? i could be automated like this.

```bash
if [[ "$authelia_local_version" != "$authelia_remote_version" ]]; then
	wget -qO "$HOME/authelia.tar.gz" "https://github.com/authelia/authelia/releases/download/v${authelia_remote_version}/authelia-linux-amd64.tar.gz"
	tar -xf "$HOME/authelia.tar.gz" -C "$HOME/bin/" authelia-linux-amd64
	mv -f "$HOME/bin/authelia-linux-amd64" "$HOME/bin/authelia"
	[[ -f "$HOME/authelia.tar.gz" ]] && rm -f "$HOME/authelia.tar.gz"
	systemctl --user restart authelia.service
	echo "Authelia updated and restarted"
fi
```

Reply:
yeah basically that, would be sweet